src/api/rest/reducers/errorMessage.js

```diff
@@ -0,0 +1,14 @@
+const RESET_ERROR_MESSAGE = 'RESET_ERROR';
```

Comment:
@lex111 in here we could store errors from the API and toast them as you mentioned in #430